dataprofiler/tests/profilers/test_base_column_options.py

```diff
@@ -1,10 +1,12 @@
 from dataprofiler.profilers.profiler_options import BaseColumnOptions
 from dataprofiler.tests.profilers.test_boolean_option import TestBooleanOption
 
-
 class TestBaseColumnOptions(TestBooleanOption):
     
-    option_class = BaseColumnOptions
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.option_class = BaseColumnOptions
```

Comment:
I think this is a safer method of setting the option_class variable. If you just set the option_class = BaseColumnOptions without the setUpClass method, it would mean that if anyone extended the TestBaseColumnOptions class (i.e. the TestNumericalOptions class) but forgot to specify the option_class, it would default to BaseColumnOptions instead of None. The desired behavior is None as the Abstract Class has a check for that.

Reply:
The reason we changed it was because internally tests were failing when done in parallel.

This created a competing setup for who is assigning the class value and resulted in it being set to None and the abstract check for the option_class value failed.

Reply:
Does this mean that any class variable would have to be defined outside of the setUpClass method (i.e. keys)?